mtags/src/main/scala/scala/meta/internal/mtags/MtagsIndexer.scala

```diff
@@ -43,6 +43,8 @@ trait MtagsIndexer {
   private var myLastCurrentOwner: String = currentOwner
   def lastCurrentOwner: String = myLastCurrentOwner
 
+  def maybeAllIdentifiers: Set[String] = Set.empty
```

Comment:
```suggestion
  def allIdentifiers: Set[String] = Set.empty
```
we usually use maybe for Option